src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -98,14 +92,15 @@ const useGroupInfo = () => {
       const sendData = {
         user_id: selectedUser.user_id
       };
+      const members = [];
+      members.push(profile?.myProfile?.user_id, selectedUser?.user_id);
       const processGetBlock = await checkUserBlock(sendData);
-
       if (!processGetBlock.data.data.blocked && !processGetBlock.data.data.blocker) {
-        return openChatMessage();
+        return createSignChat(members, selectedUser);
       }
       return handleOpenProfile(selectedUser);
     } catch (e) {
-      console.log(e, 'eman');
+      console.log(e, 'erro');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a potential issue with error handling here. In the `catch` block, you're logging the error to the console but not doing anything else with it. This could lead to silent failures that are hard to debug in production. Consider throwing the error or handling it in a way that's appropriate for your application.
```diff
-      console.log(e, 'erro');
+      throw e;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->